src/containers/Categories/Categories.js

```diff
@@ -1,27 +1,44 @@
 import React, { Component } from "react";
 import Category from "../../components/Category/Category";
+import PropTypes from 'prop-types';
 
 import "./categories.css";
 
-import MockCategories from  "../../mockdata/categories.json";
-
 class Categories extends Component {
-  state = MockCategories;
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      jokesCategories: [],
+    }
+  }
+
+  componentWillMount() {
+    fetch( 'https://api.chucknorris.io/jokes/categories' )
+      .then( response => response.json() )
+      .then( data => this.setState({jokesCategories: data}) )
+  }
 
   render() {
+    const { jokesCategories } = this.state;
     let categories = null;
+
     categories = (
```

Comment:
Could be changed to `const categories = ...` and previous variable declaration could be removed